### ISSUE TITLE
Logo For Evidence Action- Safe Water Dispensers

### ISSUE THREAD

**Issue by ascalamogna** [evidenceaction/DataDashboard-Dispensers#15]
@katrinskaya 

Just want to confirm that this is the correct logo to use on the site. If it is, do you have a horizontal version of this logo? I'm thinking it might better fill the space if it's available, if not no worries. 

Thanks!

<img width="1177" alt="screen shot 2016-03-28 at 3 40 10 pm" src="https://cloud.githubusercontent.com/assets/6752098/14088110/a317f8a6-f4fb-11e5-96d0-032a1cb9b744.png">

**Comment by katrinskaya** [comment]
No, we do not - we discontinued the horizontal logo. You can use the logo for Evidence Action without Dispensers which will make it less weird-looking. Dispensers for Safe Water is the name of the program and it's the title - so we should be good with our plain / non-program logo. 
![ealogo_red](https://cloud.githubusercontent.com/assets/658207/14102401/5bab3452-f568-11e5-95a9-5daefc35ea73.png)
